completion.go

```diff
@@ -97,7 +101,8 @@ func (c *Client) CreateCompletion(
 	ctx context.Context,
 	request CompletionRequest,
 ) (response CompletionResponse, err error) {
-	if request.Model == GPT3Dot5Turbo0301 || request.Model == GPT3Dot5Turbo {
+	switch request.Model {
+	case GPT3Dot5Turbo0301, GPT3Dot5Turbo, GPT4, GPT40314, GPT432K0314, GPT432K:
```

Comment:
Are we sure that GPT4 is not available for the `CreateCompletion` endpoint?

Reply:
I am uncertain at this time as my account does not yet have the necessary permissions to use the new GPT-4 API

Reply:
Got it, thanks! I appreciate the effort to keep the API less brittle — let's wait a bit until we know these limitations for sure.

We can either keep this PR open or close it until next time.